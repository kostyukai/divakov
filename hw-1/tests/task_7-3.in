35235235658.2354
