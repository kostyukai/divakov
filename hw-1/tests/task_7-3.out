01000000001000101000001101000010111100000111010010100011110001001010001111000100
